ios/engine/KMEI/KeymanEngine/Classes/KeymanHosts.swift

```diff
@@ -64,9 +64,16 @@ public enum KeymanHosts {
     if useLocal {
       return URL.init(string: "http://keyman.com.local")!
     } else {
-      // TODO:  Once the staging-site changes are in place,
-      //        add switch-case here to select appropriate URL.
-      return URL.init(string: "https://keyman.com")!
+      // Necessary for development of the keyboard search, even though the staging site
+      // URL isn't quite stable yet.
+      switch forTier {
+        case .alpha:
+          fallthrough
+        case .beta:
+          return URL.init(string: "https://staging-keyman-com.azurewebsites.net")!
```

Comment:
I think keyman-staging.com is now stable 😄

Reply:
In that case, I might just slip in the necessary `switch-case`s for all three... assuming that holds for all 3?  Or just keyman-staging.com?

Reply:
`keyman-staging.com` is replacing `staging-keyman-com.azurewebsites.net`.

Reply:
Right.  What about `api.keyman-staging.com` and `help.keyman-staging.com`?  Are those ready now, or do they come later?

Reply:
> What about `api.keyman-staging.com` and `help.keyman-staging.com`? Are those ready now, or do they come later?

Yes, they are ready now as well.